pure.zsh

```diff
@@ -453,8 +460,11 @@ prompt_pure_setup() {
 	# show username@host if root, with username in white
 	[[ $UID -eq 0 ]] && prompt_pure_username='%F{white}%n%f%F{242}@%m%f'
 
+	# if a virtualenv is activated, display it in grey
+	PROMPT='%(12V.%F{242}${psvar[12]}%f .)'
```

Comment:
I would prefer if we used `%12v` instead of `${psvar[12]}`. Mostly for consistency.